integration/Makefile

```diff
@@ -38,7 +38,8 @@ consensus-tests:
 
 .PHONY: epochs-tests
 epochs-tests:
-	go test $(if $(VERBOSE),-v,) $(RACE_FLAG) $(if $(JSON_OUTPUT),-json,) $(if $(NUM_RUNS),-count $(NUM_RUNS),) -tags relic ./tests/epochs/...
+	# Use a higher timeout of 20m for the suite of tests which span full epochs
+	go test $(if $(VERBOSE),-v,) $(RACE_FLAG) $(if $(JSON_OUTPUT),-json,) $(if $(NUM_RUNS),-count $(NUM_RUNS),) -tags relic -timeout 20m ./tests/epochs/...
```

Comment:
We have observed timeout failures in the past. The timeout is per test binary (package), not per test, and the epoch suite when passing is near enough to the default 10 minute timeout that the timeout failures appear to be real timeouts, rather than timeouts as a symptom of some other problem. For example, a recent passing test run (after increasing the test timeout) with 3 epoch tests enabled takes 13m for the `Run tests` step: https://github.com/onflow/flow-go/actions/runs/3596970055/jobs/6058931839